lua/wire/client/sound_browser.lua

```diff
@@ -87,6 +87,18 @@ local function GetInfoTable(strfile)
 	return T, not tabproperty
 end
 
+local function GetGamesForSound(path)
+	local games = {}
+
+	for _,game in pairs(engine.GetGames()) do
+		if not game.mounted then return end
+		if not file.Exists(path, game.folder) then return end
```

Comment:
oops, you're right. I had previously only returned one value :)
I'll fix it right away